src/components/AccountDetails/index.tsx

```diff
@@ -433,75 +432,24 @@ export default function AccountDetails({
                     verticalAlign: 'middle',
                   }}
                 >
-                  <WalletAction
-                    style={{ fontSize: '14px', fontWeight: 600 }}
-                    onClick={() => {
-                      openOptions()
-                    }}
-                  >
-                    <Trans>Change Wallet</Trans>
-                  </WalletAction>
                   {connector !== injected && connector !== walletlink && (
                     <WalletAction
                       style={{ fontSize: '14px', fontWeight: 600 }}
-                      onClick={() => {
-                        ;(connector as any).close()
-                      }}
+                      onClick={() => (connector as any).close()}
                     >
                       <Trans>Disconnect Wallet</Trans>
                     </WalletAction>
                   )}
                 </div>
               </AccountGroupingRow>
             </InfoCard>
+            <div style={{ padding: '10px', justifyContent: 'space-around' }}>
+              <button style={{ width: '48%', height: '20px' }}>Disconnect Wallet</button>
+              <button style={{ width: '48%', height: '20px' }}>Explorer</button>
+            </div>
           </YourAccount>
         </AccountSection>
       </UpperSection>
```

Comment:
May I know what is the part below, that you have deleted? What is it related to in the UI? @raynear

Reply:
below thing is about transaction history. it's not on our new design.